### PR TITLE
[Behat] Extract javascript driver check

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.11 |
| Bug fix?        | no |
| New feature?    | no  |
| BC breaks?      | no  |
| Deprecations?   |  |
| Related tickets |   |
| License         | MIT |

It was killing me for ages :dancer:

### CODE REVIEW

**Review by NoResponseMate — APPROVED** [review]

File: src/Sylius/Behat/Service/DriverHelper.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Service;
+
+use Behat\Mink\Driver\DriverInterface;
+use Behat\Mink\Driver\Selenium2Driver;
+use DMore\ChromeDriver\ChromeDriver;
+
+abstract class DriverHelper
```
Comment (lchrusciel) [inline src/Sylius/Behat/Service/DriverHelper.php:11]: what about
```suggestion
final class DriverHelper
```
with a private constructor, so the class won't be initializable?

**Review by lchrusciel — APPROVED** [review]
I'm wondering if an injection of this service won't be a better way to solve it, so the 3rd party contributors would be able to use different drivers within. On the other hand, the external DX stays the same (you are still not able to change these drivers easily), but it boosting our DX and reduce possible maintenance issues with changing drivers everywhere manually, so all in all, big 👍

File: src/Sylius/Behat/Service/DriverHelper.php (hunk 1 above)
Comment (Zales0123) [reply src/Sylius/Behat/Service/DriverHelper.php:11]: I've used the same construction as in `JQueryHelper` but I agree, we could change them both 👍

**Comment by loic425** [comment]
@Zales0123 great Idea, maybe it's Time to finish panther support I started months ago

**Pull request merged**